tests/integration/test_deploy_model_ssl_on_auth_on.py

```diff
@@ -19,10 +18,12 @@ def _get_pwd_file(self) -> str:
         return './tests/integration/resources/pwdfile.txt'
 
     def test_deploy_ssl_on_auth_on(self):
-        models = ['PCA', 'Sentiment%20Analysis', "ttest"]
-        path = str(Path('models', 'setup.py'))
-        p = subprocess.run([self.py, path, self._get_config_file_name()],
-                           input=b'user1\nP@ssw0rd\n')
+        # Uncomment the following line to preserve
+        # test case output and other files (config, state, ect.)
+        # in system temp folder.
+        # self.set_delete_temp_folder(False)
```

Comment:
do we care about this being customizable? surely we're not going to advertise to users that they install Python via pip, navigate into site packages, and then uncomment a line of code in order to preserve temp files

Reply:
Correct